spec/mailers/audit_mailer_spec.rb

```diff
@@ -0,0 +1,59 @@
+require "rails_helper"
+
+RSpec.describe AuditMailer, type: :mailer do
+  let(:audit_user) { double(:audit_user, email: Faker::Internet.email) }
```

Comment:
Note: The use of `audit.user.mailer` in the AuditMailer forces the creation of this tree of objects.